recipe/build.sh

```diff
@@ -1,7 +1,10 @@
 #!/bin/bash
 
-./configure --prefix=$PREFIX
+export CFLAGS="-I$PREFIX/include -L$PREFIX/lib -O2 -Wl,-S $CFLAGS"
```

Comment:
This stuff really shouldn't be necessary.  Not sure specifically about -Wl,-S, but everything else is already in both the anaconda compilers and toolchain.

Reply:
I understand that and I'll remove once this works (I need this for... reasons...). However, we need to discuss the inclusion of `-Wl,-S`, or maybe even more aggressive stripping, to reduce the binary size of the packages.

Final `fiona` and `rasterio` conda packages used to be much smaller than the corresponding wheels, but since they started stripping all the gdal dependencies now the conda version of those packages is quite larger in comparison.